signer-cloud-server/src/main/java/com/wultra/signercloud/server/signer/IssuedCertificateRepository.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Signer Cloud
+ * Copyright (C) 2025 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.signercloud.server.signer;
+
+import org.springframework.data.jdbc.repository.query.Query;
+import org.springframework.data.repository.CrudRepository;
+
+import java.time.Instant;
+import java.util.List;
+
+/**
+ * Repository for accessing a {@link IssuedCertificate}.
+ *
+ * @author Michal Rozehnal, michal.rozehnal@wultra.com
+ */
+interface IssuedCertificateRepository extends CrudRepository<IssuedCertificate, Long> {
+
+    @Query("""
+        SELECT * FROM sc_issued_certificate
+        WHERE signer_id = :signerId AND timestamp_certificate_expiration < :now
```

Comment:
No strong opinion, but we may use `NOW()` and avoid passing the date from the Java code.
```suggestion
        WHERE signer_id = :signerId AND timestamp_certificate_expiration < NOW()
```

Reply:
I used the `NOW()` in order to be consistent